### PR TITLE
Home page ui

### PR DESCRIPTION
**Pull request by anagabriel42**

### CODE REVIEW

File: app/views/material_admin/home.html.erb [hunk 1]
```diff
@@ -1,2 +1,87 @@
...
+                  <img src="" alt="">
+              </div>
+
+              <div class="sp-info">
+                  Malinda Hollaway
+                  <i class="material-icons">expand_more</i>
+              </div>
+          </a>
+
+          <ul class="main-menu">
```
Comment (anagabriel42) [inline app/views/material_admin/home.html.erb]: this is a part you need to do checks on

File: app/views/material_admin/home.html.erb [hunk 2]
```diff
@@ -1,2 +1,87 @@
...
+              <li>
+                  <a href="">
+                    <i class="material-icons">restore</i>
+                    Logout
+                  </a>
+              </li>
+          </ul>
+      </div>
+
+      <ul class="main-menu">
```
Comment (anagabriel42) [inline app/views/material_admin/home.html.erb]: and this one

File: app/views/material_admin/home.html.erb [hunk 3]
```diff
@@ -1,2 +1,87 @@
...
+  <div class="h-search-wrap">
+      <div class="hsw-inner">
+          <i class="material-icons hsw-close" data-ma-action="search-close">arrow_back</i>
+          <input type="text">
+      </div>
+  </div>
+</header>
+
+<section id="main">
+  <aside id="sidebar" class="sidebar c-overflow">
```
Comment (anagabriel42) [inline app/views/material_admin/home.html.erb]: this part might need to be customized a little more, because we haven't talked about storing images yet

**Comment by sayerwalam52** [comment]
+1

**Pull request merged**